native/src/main/java/io/ballerina/stdlib/http/api/HttpCallableUnitCallback.java

```diff
@@ -78,8 +79,9 @@ public void notifyFailure(BError result) {
                 sendFailureResponse(result);
             }
         };
-        runtime.invokeMethodAsync(caller, "returnResponse", null, ModuleUtils.getNotifySuccessMetaData(),
-                                  returnCallback, paramFeed);
+        runtime.invokeMethodAsyncConcurrently(
```

Comment:
Why is it related to making the caller isolated ?

Reply:
This runtime API change ([#2383](https://github.com/ballerina-platform/ballerina-standard-library/issues/2383)) has not covered the HTTP resource return scenario. Hence fixed it with isolation change because after marking the `http:Caller` isolated, we could call remote methods concurrently checking whether it is isolated.

Reply:
As per the discussion, changed this to use `invokeMethodAsyncSequentially` with [6b31ce46165cd8fec531377334a9aabcd9d497e8](https://github.com/ballerina-platform/module-ballerina-http/pull/765/commits/6b31ce46165cd8fec531377334a9aabcd9d497e8)